StarWars5e.Parser/Parsers/MonsterManualProcessor.cs

```diff
@@ -222,70 +227,75 @@ private Monster ParseMonster(List<string> monsterLines)
                     NumberStyles.AllowThousands, null,  out var parsedXP);
                 monster.ExperiencePoints = didParseXP ? parsedXP : 0;
 
-                monster.Behaviors = new List<MonsterBehavior>();
-                var lastUnderScoreLine = monsterLines.FindLastIndex(f => f.Contains("___")) + 1;
-                var firstTripleHash = monsterLines.FindIndex(f => f.StartsWith("> ###"));
+                ParseBehaviorLines(monster, monsterLines);
+
+                return monster;
+            }
+            catch (Exception e)
+            {
+                throw new Exception($"Failed while parsing {name}", e);
+            }
+        }
+
+        private void ParseBehaviorLines(Monster monster, List<string> monsterLines)
+        {
+            var firstTripleHashIndex = monsterLines.FindIndex(f => f.StartsWith("> ###"));
+            var lineAfterLastUnderscoreIndex = monsterLines.Select((line, index) => new { line, index })
+                .Last(s => s.line.Contains("___") && s.index < firstTripleHashIndex).index + 1;
```

Comment:
the previous logic was simply retrieving the last occurrence of "____", but I think it was intended to also be of lesser index than the `firstTripleHashIndex`